packages/design-tokens/package.json

```diff
@@ -30,7 +30,7 @@
     "build:js": "vite build",
     "build:scss": "shx mkdir -p scss && shx cp -r src/scss/generated/* src/scss/@tokens.scss scss/",
     "clean": "rimraf esm cjs umd scss types",
-    "lint": "stylelint ./src/**/*.scss",
+    "lint": "eslint ./src/**/*.ts && stylelint ./src/**/*.scss",
```

Comment:
`"lint": "npm-run-all --serial lint:scripts lint:styles"`

Reply:
ERROR: Task not found: "lint:scripts", lint:styles"